app/_layouts/product.njk

```diff
@@ -30,19 +30,32 @@
       {% endif %}
     {# List sections (pages with a parent that is the homepage) if homepage has no pagination key set #}
     {% elif eleventyNavigation.key == options.homeKey %}
-      <div class="govuk-grid-column-full">
-        <h2 class="govuk-heading-l govuk-!-font-size-27">
-          {{ sections.title or app.sections.title or "Sections" }}
-        </h2>
-      </div>
-      {% for item in collections.all | eleventyNavigation(options.homeKey) %}
-        <section class="govuk-grid-column-one-half govuk-!-margin-bottom-6">
-          <h3 class="govuk-heading-m govuk-!-margin-bottom-1">
-            <a href="{{ item.url | pretty }}">{{ item.title }}</a>
-          </h3>
-          <p class="govuk-body">{{ item.excerpt }}</p>
-        </section>
+
+      {% set sections = [
+      {title: "Screening", services:["Bowel screening", "Explore team", "Manage breast screening"]}
```

Comment:
Indent and add a bit more whitespace?

Not ideal to need to hardcode these names here - they already exist hardcoded in several places - but I guess we can tackle that later. We should probably write a mini readme on what you need to do to add a service to this design history as there's several files you need to update.

Reply:
Added whitespace and updated README, thanks.